src/stores/search.ts

```diff
@@ -4,7 +4,9 @@ import { defineStore } from 'pinia'
 // https://github.com/WordPress/openverse-frontend/issues/1103
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import clonedeep from 'lodash.clonedeep'
+import { deepclone } from '~/utils/clone'
```

Comment:
We can remove the ts-ignore and the comments on lines 4-6! 🙂

Reply:
Oops, I removed them from the other one :facepalm: Thanks for catching this one!